msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Designing your meeting extension
 author: heath-hamilton
-description: Learn how to design meeting extensions for your apps in Teams meetings. Use the UI templates in the Microsoft Teams UI Kit to help you design your meeting tab.
+description: Learn how to implement the design guidelines and use the UI templates to design a meeting extension for Teams.
```

Comment:
I doubt if learning the design guidelines is the core focus of the article. If not, then don't frontload it in the description. I think the core focus of this article is creation of an extn. using the design guidance. In this case, flip this sentence.

Reply:
Edited and updated the description